README.md

```diff
@@ -0,0 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Description the project.
+
+
+## Built With
+
+- HTML CSS
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
```

Comment:
- [ ] I would recommend you write how a user of your repo can get started by highlighting the steps it takes to get the local copy of the website to the user's local machine 😄

Reply:
I did not get your point, please explain it in an easy way

Reply:
State how to clone the website and how to navigate to the project working directory
E.g

step 1
git clone `link of the repo`
cd `working directory`